Dockerfile

```diff
@@ -65,8 +65,12 @@ RUN chmod +x /usr/local/bin/optimize.sh
 
 ADD optimize_workspace.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/optimize_workspace.sh
-ADD build_workspace.py /usr/local/bin/
-RUN chmod +x /usr/local/bin/build_workspace.py
+
+ADD build_workspace build_workspace
+ENV RUSTFLAGS="-C link-arg=-s"
```

Comment:
Is this really necessary? How large is `build_workspace`? I'd prefer sticking to solid defaults for native builds.

Reply:
500K vs 688K, so no it's not really necessary :) Will remove

Reply:
Thanks. Not worth thinking about it in that case.